bindata/etcd/cluster-backup.sh

```diff
@@ -54,17 +65,17 @@ BACKUP_TAR_FILE=${BACKUP_DIR}/static_kuberesources_${DATESTRING}.tar.gz
 SNAPSHOT_FILE="${BACKUP_DIR}/snapshot_${DATESTRING}.db"
 BACKUP_RESOURCE_LIST=("kube-apiserver-pod" "kube-controller-manager-pod" "kube-scheduler-pod" "etcd-pod")
 
-trap "rm -f ${BACKUP_TAR_FILE} ${SNAPSHOT_FILE}" ERR
+trap 'rm -f ${BACKUP_TAR_FILE} ${SNAPSHOT_FILE}' ERR
```

Comment:
This seems like an important change — was it broken somehow because of the expansion?

Reply:
It works either way in this case. With double quotes the expansion happen at the definition of trap, whereas with single quote the expansion happens when signalled.

I think Sam ran through shellcheck and fixing the errors and warnings it reports.

Reply:
For example, consider this test:
```
#!/bin/bash
tempfile="/tmp/tmpdata1"
trap 'echo $tempfile' EXIT
tempfile=/tmp/tmpdata2
```
If double quote is used, it prints `/tmp/tmpdata1`, but if single quote is used it prints `/tmp/tmpdata2`. But in our case the variables are defined once and never changed, so it doesn't matter.